text/0000-deprecate-ember-logger.md

```diff
@@ -0,0 +1,265 @@
+
+There are a variety of features of Ember designed to support old browsers,
+features that are no longer needed. Ember.Logger came into being because
+the browser support for the console was inconsistent. In some browsers,
+like Internet Explorer 9, the console only existed when the developer tools
+panel was open, which caused references to the console object's methods to
+fail to execute. Ember.Logger provided methods that would route to the 
+console when it was available.
+
+With Ember 3.x, Ember no longer supports these older browsers, and hence this
+feature no longer serves a purpose. Removing it will make Ember smaller and 
+lighter.
+
+## Detailed design
+
+### For framework developers
+
+Remove the following 8 direct uses of Ember.Logging from ember.js and 
+ember-data: 
+
+ember-debug:
+*  deprecate (deprecate.js) - Logger.warn
+*  debug (index.js) - Logger.info
+*  warn (warn.js) - Logger.warn
+ember-routing (router.js):
+*  transitioned to - Logger.log
+*  preparing to transition to - Logger.log
+*  intermediate-transitioned to - Logger.log
+ember-testing:
+*  Testing paused (pause_test.js) - Logger.info
+*  Catch-all handler (adapter.js) - Logger.error
+ember-data:
+*  tests\test-helper.js:      Ember.Logger.log(reason, reason.stack);
+(Note: I found one usage in ember-data, but emberobserver claims two more.)
+
+Adjust all test code that redirects logging and sets it back:
+
+ember:
+*    routing: basic_test.js (adjust)
+ember-application - system: 
+*    logging_test.js (remove?)
+*    dependency_injection : default_resolver.test.js (adjust)
+ember-glimmer - integration: 
+*    helpers: log-test.js (remove)
+
+None of the uses of Ember.Logging in ember.js or ember-data are using 
+Ember.debug, which will simplify matters.
+
+### For Framework Users
+
+For the most part, this is a 1:1 substitution of the global console object 
+for Ember.Logging. The only anomaly remaining in Javascript engine console 
+support is that versions of Node earlier than Node 9 do not support 
+console.debug, so console.log will need to be used as the replacement for 
+Logging.debug, rather than console.debug, unless the minimum Node support 
+level is set to Node 9.
+
+The following high-impact packages (9 or 10 or a * on EmberObserver) are 
+calling Ember.Logging and should probably be given an early heads-up to 
+adjust their code to use console in advance of the release containing the 
+deprecation to limit the level of pain that their users experience. In 
+the order of their reliance on Ember.Logging:
+
+* ember-concurrency (15)
+* ember-cli-deprecation-workflow (9)
+* ember-stripe-service (9)
+* semantic-ui-ember (7)
+* ember-resolver (6)
+* ember-cli-page-object (4) 
+* ember-cli-sentry (3)
+* ember-islands (3)
+* ember-states (3)
+* ember-cli-pagination (2)
+* ember-cli-clipboard (1)
+* ember-cli-fastboot (1)
+* ember-elsewhere (1)
+* ember-i18n (1)
+* ember-simple-auth-token (1)
+* ember-svg-jar (1)
+* liquid-fire (1)
+
+### Timeline
```

Comment:
Hm, I think this section might be better off being merged into the two previous ones.